### PR TITLE
:zap: [RND-49612] Add dark theming for FieldLabel

### PR DESCRIPTION
**Pull request by chaeMil**
#### Summary

*Describe anything that you feel other developers should know.*

#### Testing steps

*Info about test cases. If you think the issue is not testable, describe why.*

#### Follow-up issues

*Is there some action/cleanup needed after this PR is merged or deployed (e.g. consolidate some part outside the scope of this PR, etc)? Create issues for it with deadline and note them here and in the PR comments.*

#### Check during review

- Verify against YouTrack issue.
- Is the PR over 300 additions? Consider rejecting it with advice to split it. Is it over 500 additions? It should definitely be rejected.
- Unused code removed.
- Build passing.
- Is it a bug fix? Check that it is covered by a proper test (unit or integration).

### CODE REVIEW

File: optimus/lib/src/field_label.dart [hunk 1]
```diff
@@ -13,11 +15,24 @@ class OptimusFieldLabel extends StatelessWidget {
...
-      );
+  Widget build(BuildContext context) {
+    final theme = OptimusTheme.of(context);
+
+    return Padding(
+      padding: const EdgeInsets.only(bottom: 2),
+      child: OptimusLabelSmall(
+        variation: Variation.variationSecondary,
+        child: Text(isRequired ? '$label *' : label,
+            style: TextStyle(color: _textColor(theme))),
```
Comment (ookami-kb) [inline optimus/lib/src/field_label.dart:26]: Nitpicking: add trailing comma.

File: optimus/lib/src/field_label.dart [hunk 2]
```diff
@@ -13,11 +15,24 @@ class OptimusFieldLabel extends StatelessWidget {
...
+      ),
+    );
+  }
+
+  Color _textColor(OptimusThemeData theme) {
+    if (theme.isDark) {
```
Comment (ookami-kb) [inline optimus/lib/src/field_label.dart:32]: Nitpicking: I would go with ternary operator.

**Comment by chaeMil** [comment]
@ookami-kb Also added support for `GroupWrapper`

**Review by Gladkowski — APPROVED** [review]

**Comment by ookami-kb** [comment]
@chaeMil I've pushed some update to re-use typography components.

**Review by ookami-kb — APPROVED** [review]

**Pull request merged**